### PR TITLE
Create Unpackaged publish docs for Windows

### PR DESCRIPTION
**Pull request by mattleibow**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/windows/deployment/publish-cli.md](https://github.com/dotnet/docs-maui/blob/40fa77b881fb37ac8a5bf09ad733a06c852a5083/docs/windows/deployment/publish-cli.md) | [Publish a .NET MAUI app for Windows with the CLI](https://review.learn.microsoft.com/en-us/dotnet/maui/windows/deployment/publish-cli?branch=pr-en-us-1686) |
| [docs/windows/deployment/publish-unpackaged-cli.md](https://github.com/dotnet/docs-maui/blob/40fa77b881fb37ac8a5bf09ad733a06c852a5083/docs/windows/deployment/publish-unpackaged-cli.md) | [docs/windows/deployment/publish-unpackaged-cli](https://review.learn.microsoft.com/en-us/dotnet/maui/windows/deployment/publish-unpackaged-cli?branch=pr-en-us-1686) |

### CODE REVIEW

**Comment by mattleibow** [comment]
This PR should probably land first so the assets are copied. https://github.com/dotnet/maui/pull/17238

The IDE works fine, just the publish steps are not working correctly.

File: docs/windows/deployment/publish-cli.md [hunk 1]
```diff
@@ -105,7 +103,7 @@ Add the following `<PropertyGroup>` node to your project file. This property gro
 
 Replace the `<PackageCertificateThumbprint>` property value with the certificate thumbprint you previously generated. Alternatively, you can remove this setting from the project file and provide it on the command line. For example: `-p:PackageCertificateThumbprint=A10612AF095FD8F8255F4C6691D88F79EF2B135E`.
 
-The second `<PropertyGroup>` in the example is required to work around a bug in the Windows SDK. For more information about the bug, see [WindowsAppSDK Issue #2940](https://github.com/microsoft/WindowsAppSDK/issues/2940).
+The second `<PropertyGroup>` in the example is required to work around a bug in the Windows SDK. For more information about the bug, see [WindowsAppSDK Issue #3337](https://github.com/microsoft/WindowsAppSDK/issues/3337).
```
Comment (MSLukeWest) [inline docs/windows/deployment/publish-cli.md]: nit: "Windows App SDK", not "Windows SDK"

File: docs/windows/deployment/publish-unpackaged-cli.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Add the following `<PropertyGroup>` node to your project file. This property group is only processed when the target framework is Windows and the configuration is set to `Release`. This config section runs whenever a build or publish in `Release` mode.
+
+```xml
+<PropertyGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows' and '$(RuntimeIdentifierOverride)' != ''">
+    <RuntimeIdentifier>$(RuntimeIdentifierOverride)</RuntimeIdentifier>
+</PropertyGroup>
+```
+
+The `<PropertyGroup>` in the example is required to work around a bug in the Windows SDK. For more information about the bug, see [WindowsAppSDK Issue #3337](https://github.com/microsoft/WindowsAppSDK/issues/3337).
```
Comment (MSLukeWest) [inline docs/windows/deployment/publish-unpackaged-cli.md]: Same as above, say "Windows App SDK" instead

**Review by MSLukeWest — APPROVED** [review]

**Comment by davidbritch** [comment]
I thought MAUI lacked full support for unpackaged apps? e.g. https://github.com/dotnet/maui/issues/10564, https://github.com/dotnet/maui/issues/8553, https://github.com/dotnet/maui/issues/8552

File: docs/windows/deployment/publish-unpackaged-cli.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+> [!WARNING]
+> Attempting to publish a .NET MAUI solution will result in the `dotnet publish` command attempting to publish each project in the solution individually, which can cause issues when you've added other project types to your solution. Therefore, the `dotnet publish` command should be scoped to your .NET MAUI app project.
+
+For example:
+
+```console
+dotnet publish -f net7.0-windows10.0.19041.0 -c Release -p:RuntimeIdentifierOverride=win10-x64 -p:WindowsPackageType=None
+```
+
+Publishing builds and packages the app, copying the signed package to the _bin\\Release\\net7.0-windows10.0.19041.0\\win10-x64\\AppPackages\\\<appname>\\_ folder. \<appname> is a folder named after both your project and version. In this folder, there's an _msix_ file, and that's the app package.
```
Comment (MSLukeWest) [inline docs/windows/deployment/publish-unpackaged-cli.md]: This line seems wrong.  This leads the user to the location where we put _packaged_ apps, it even mentions that there should be an msix file (a package).  The actual location the unpackaged publish goes to is something like: "MauiApp30\bin\Release\net8.0-windows10.0.19041.0\win10-x64\publish".  And when I try to directly start the exe in there nothing happens.  So, I wouldn't consider this a correctly unpackaged app.

**Comment by MSLukeWest** [comment]
Suggestion: Mention the MSBuild property you'd pass to dotnet publish to change from framework dependent (the default) to self-contained (where all necessary runtime binaries are copied into the app's folder).  This would be "-p:SelfContained=True".

File: docs/windows/deployment/publish-unpackaged-cli.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+> [!div class="op_single_selector"]
+>
+> - [Publish Packaged / MSIX](publish-cli.md)
+> - [Publish Unpackaged / EXE](publish-unpackaged-cli.md)
+
+When distributing your .NET Multi-platform App UI (.NET MAUI) app for Windows, you can publish the app and its dependencies to a folder for deployment to another system.
+
+## Configure the project build settings
+
+The project file is a good place to put Windows-specific build settings. You may not want to put some settings into the project file, such as passwords. The settings described in this section can be passed on the command line with the `-p:name=value` format. If the setting is already defined in the project file, a setting passed on the command line overrides the project setting.
```
Comment (BretJohnson) [inline docs/windows/deployment/publish-unpackaged-cli.md]: Should we remove this paragraph for unpackaged, to simplify the steps? For unpackaged, there are no passwords. And I don't see any instructions below that give other settings that should go in the project file (except for RuntimeIdentifier, which I assume has to go there).

**Comment by davidbritch** [comment]
Replaced by #1864

**Pull request closed without merging**